tpcx_bb/benchmark_runner/draco_config.yaml

```diff
@@ -0,0 +1,21 @@
+# benchmark config yaml
+### Please fill these accordingly
+data_dir: /tpcx-bb/sf1000/parquet_3gb_test/
+output_dir: ./
```

Comment:
Is this where we want to be writing to on Draco? Do we want to create local paths on each node to some kind of local storage?